news/serialization-harmonization.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* ``GufeTokenizable.from_json`` now falls back to loading ``dict`` representation if from ``keyed_chain`` fails
+
+**Deprecated:**
+
+* ``Transformation.dump``, ``Transformation.load`` are now deprecated
+
+**Removed:**
+
+* ``ExplicitMolecule.to_json``, ``ExplicitMolecule.from_json`` removed; now uses ``GufeTokenizable`` methods
```

Comment:
they still have these methods though, correct? theyre just inherited from `GufeTokenizable`? If so, I'd consider this a "change"

Reply:
Done!